timescale-forge/disk-management.md

```diff
@@ -0,0 +1,112 @@
+
+So that you aren't overwhelmed by automated message, the alerting thresholds use
+low- and high-watermarks, and only one message is sent per database per day.
+
+## Automated overload protection
+
+If your database continues to increase in size past these thresholds, automated
+overload protection is activated when your disk becomes 99% full. When this
+happens, your database is put into read-only mode, and you receive a
+notification on email and the Timescale Forge console shows the changed status.
+
+When this happens, you can still query your database, but you cannot add any new
+data to it. This is to ensure that your disk does not fill up to 100%, which
+could prevent you from recovering the database.
+
+With your database in read-only mode, you need to decide if you are going to
+increase your storage capacity, or reduce the size of your database.
+
+## Online storage resizing
+
+You can increase your storage size in the Timescale Forge console.
+
+<highlight type="warning">
+You can only resize your services once every six hours.
+</highlight>
+
+### Procedure: Increasing service resources
+1.  In the Timescale Forge console, navigate to `Services` and click the service
+you want to adjust. Navigate to the `Operations` tab, and go to the `Resources`
+section.
+1.  Adjust the sliders for CPU and disk size as required. If you increase the
+disk size past a certain point, you may also need to increase the CPU size to
+handle the increased disk size.
+1.  Review the new sizes and costs in the panel on the right-hand side, and
+click `Restart and apply` when you are happy with the changes.
+1.  The resources take a few seconds to increase, and when the increase is
+complete, your database is immediately available on the new resources. There is
+no downtime associated with this action. If your database is in read-only mode,
+the read-only protection is automatically removed, and you can begin writing
+data immediately.
+
+<img class="main-content__illustration" src="https://assets.timescale.com/images/diagrams/forge_resources.png" alt="Timescale Forge change resources"/>
+
+## Storage recovery
+
+If you need to perform actions on your database to reduce your data usage, you
+can turn off read-only mode. For example, you need read-write access if you want
+to compress data, delete rows or tables, or drop old data using data retention
+policies.
+
+<highlight type="warning"> We highly recommend that you do not manually enable
+read-write mode on a database that is over 99% capacity. You should consider
+increasing the disk size before you do this. Alternatively, you can enable
+read-write access on an individual session, while leaving the database in
+read-only mode. </highlight>
+
+### Procedure: Enabling read-write access on an individual session
+1.  Connect to your database using `psql` and turn off read-only protection for the current session:
+    ```sql
+    SET default_transaction_read_only TO off;
+    ```
+1.  Turn on compression:
```

Comment:
I think when database is in read-only it's better either to clean-up it, or to resize the volume.
Trying to enable compression at this point can lead to filling database to 100%. :smile_cat: 
cc @lukas-bernert

Reply:
thanks for the note @itroot 

So it would make sense to structure it this way:

2./3. (no particular priority) Drop older Chunks OR increase disk size

4./5. (no particular priority) Add a retention policy OR turn on compression

CC: @Loquacity